webapp/api_handlers.go

```diff
@@ -48,6 +49,9 @@ func apiTestRunsHandler(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	labels := ParseLabelsParam(r)
+	unstable := labels != nil && labels.Contains("unstable")
```

Comment:
I'm reminded of https://github.com/w3c/web-platform-tests/issues/5956 and it doesn't seem far-fetched that it might be mistaken to mean flakiness on wpt.fyi as well, or that the label name makes its way back to wpt PRs via https://github.com/w3c/web-platform-tests/issues/10503 happens.

But the only counter-proposal I can come up with which accurately describes Chrome Dev/Canary, Edge Insider Preview, Firefox Nightly and Safari Technology Preview is "latest". But if you don't think that's any better, then I surrender to "unstable" :)